docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -547,6 +547,19 @@ You can use the [`moved-lab-modules` codemod](https://github.com/mui-org/materia
   +<BottomNavigation onChange={(event: React.SyntheticEvent) => {}} />
   ```
 
+- Remove the `span` element that wraps the children. Remove the `wrapper` classKey too. More details about [this change](https://github.com/mui-org/material-ui/pull/26666).
```

Comment:
I think it's confusing linking different PR here. Let's update the current PR descirption with the details of why the changes was introduced and link this PR in the migration guide.